pkg/ton/tracetracking/utils.go

```diff
@@ -0,0 +1,30 @@
+package tracetracking
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/xssnick/tonutils-go/tlb"
+	"github.com/xssnick/tonutils-go/ton"
+
+	"github.com/smartcontractkit/chainlink-ton/pkg/ton/tvm"
+)
+
+// WaitForTrace waits for the trace of a given transaction.
+func WaitForTrace(ctx context.Context, c ton.APIClientWrapped, tx *tlb.Transaction) error {
```

Comment:
Don't see where this is used, in MCMS ? nit: in message.go we also have a WaitForTrace(), maybe rename it to WaitForTx() or WaitForTxTrace()